src/app/(proper_react)/redesign/(authenticated)/user/welcome/View.tsx

```diff
@@ -53,14 +53,10 @@ export const View = ({
     }
 
     record("page", "view", {
```

Comment:
This might be one for https://github.com/mozilla/blurts-server/pull/3913, but I just realised that, as the user moves through the flow, this just sends the same event two or three times. We might want to record `currentStep` as well somewhere?

Then again, we should be able to deduce that by looking at the event that preceded this, since we're also tracking interactions with the various CTAs in the flow.

Reply:
So technically we can determine the step from utm_source (which might still not be entirely the correct parameter) and deduce the step this event came from.